sony-aosp-l.xml

```diff
@@ -15,10 +15,11 @@
   <project name="platform/hardware/qcom/media" path="hardware/qcom/media"/>
   <project name="platform/hardware/qcom/wlan" path="hardware/qcom/wlan"/>
   <project name="platform/hardware/ril" path="hardware/ril"/>
-  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.0_r1"/>
-  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r3"/>
+  <project name="platform_external_libnfc-nci" path="external/libnfc-nci" remote="b2g" revision="b2g-5.1.1_r29"/>
+  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.1_r29"/>
+  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r29"/>
   <remove-project name="platform/hardware/libhardware"/>
-  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="sony-aosp-l"/>
+  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="b2g-5.1.1_r29"/>
```

Comment:
Hm maybe we should rename that branch? It's changing from sony-aosp-l to b2g-5.1.1_r29